catkit/emulators/iris_ao_controller.py

```diff
@@ -36,8 +37,17 @@ def __init__(self, mcf_filename, custom_flat_filename, mirror_serial, **super_kw
         self.relax()  # ??? See https://github.com/spacetelescope/catkit/issues/63 (we don't currently relax the bostons like this).
 
     def set_surface(self, new_surface):
+
+        def convert_command_to_poppy_surface(dm_command_dict):
+            # Adding the negative relaxed state to any new command in order to compensate for relaxation offset
+            # Addition of dicts taken from old JOST version here:
+            # https://github.com/spacetelescope/jost-package/blob/aa7cc5517bb474eed971db0f4007ce5fc1226fec/jost/DM_functions.py#L453
+            dm_surface = {seg: tuple(np.asarray(self.relaxed_poppy_surface.get(seg, (0., 0., 0.))) + np.asarray(dm_command_dict.get(seg, (0., 0., 0.)))) for
+                          seg in set(self.relaxed_poppy_surface) | set(dm_command_dict)}
+            return dm_surface
+
         # Setting the simulated IrisAO means setting each actuator individually
-        for seg, values in new_surface.items():
+        for seg, values in convert_command_to_poppy_surface(new_surface).items():
             self.set_actuator(seg-1, values[0] * u.um, values[1] * u.mrad, values[2] * u.mrad)  # TODO: double-check the -1 here, meant to correct for different segment names
```

Comment:
This could be improved upon but doesn't need to hold up this PR.

Reply:
agreed

Reply:
Also, there is actually a bug in this which I will fix in [CATKIT-50](https://jira.stsci.edu/browse/CATKIT-50).